core/controllers/base_test.py

```diff
@@ -307,9 +307,8 @@ def test_keys_in_source_code_match_en(self):
                         files_checked += 1
                         html_key_list = self._extract_keys_from_html_file(
                             os.path.join(root, filename))
-                        if set(html_key_list) <= set(en_key_list):
-                            pass
-                        else:
+                        #pylint: disable=unneeded-not
```

Comment:
This comment should actually go on the right of the following line (separated by two spaces). Otherwise, it disables this check not only for the following line, but for all code onwards, I think. It is OK if, in this case, the following line ends up being > 80 chars (but run the linter to check).